### ISSUE TITLE
Make COI-serviceworker optional?

### ISSUE THREAD

**Issue by davcri** [crystal-bit/godot-game-template#90]
This PR https://github.com/crystal-bit/godot-game-template/pull/89 fixed the web deploy.
However this configuration should not be the default and should be opt-in: if you are deploying a web game and you can set the appropriate HTTP headers it's better to configure them that way instead of relying on https://github.com/gzuidhof/coi-serviceworker 

See also https://github.com/godotengine/godot-proposals/issues/6616

**Comment by SecurityRisk** [comment]
I agree with this. Right now, if you run the project from within Godot, you'll get a 404 error in the browser console because this file wasn't added to the project. Making it optional or making it only get applied on GH Pages would be really useful

**Comment by davcri** [comment]
@SecurityRisk I'd like to know your opinion on #93. I left some comments there.

**Comment by davcri** [comment]
Relevant: https://godotengine.org/article/dev-snapshot-godot-4-3-dev-3/#single-threaded-web-exports
